.vscode/settings.json

```diff
@@ -5,7 +5,7 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
   "[json]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "vscode.json-language-features"
```

Comment:
I don't know what this does, but it seems like the kind of thing that, if we want it, should happen in another PR. Introducing formatting change on top of a big PR like this can make it hard to find the relevant parts.

Reply:
This was fixed in telmo's branch, did you merge it in yours @nanocryk

Reply:
Yes I merged it, but going back and forth through master showed all the diffs again. Let me fix that.